### PR TITLE
[Keymap] narze/xd004

### PR DESCRIPTION
**Pull request by narze**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add narze keymap of keyboard xd004

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd004/keymaps/narze/keymap.c [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    // 0: Base Layer
+    LAYOUT_all(HYPR(KC_F2), HYPR(KC_F1), LCTL(KC_B), LCTL(KC_X)),
+
+};
+
+// Loop
+void matrix_scan_user(void){
+    // Empty
+};
```
Comment (fauxpark) [inline keyboards/xd004/keymaps/narze/keymap.c:13]:
```suggestion
```

File: keyboards/xd004/keymaps/narze/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- Key 1 : Hyper + F2
+- Key 2 : Hyper + F1
+- Key 3 : Ctrl + B (Microphone : Toggle mute)
+- Key 4 : Ctrl + X (Microphone : Push to talk)
+
+## Build & Flash
+
+- Put this board in DFU : bridge the last two pins (#5 & 6) of the 6-pin "J2" header on the opposite edge of the board to the USB port.
+- `make xd004/v1:narze:dfu` or `qmk flash -kb xd004/v1 -km narze`
```
Comment (fauxpark) [inline keyboards/xd004/keymaps/narze/readme.md:13]:
```suggestion
- `make xd004/v1:narze:flash` or `qmk flash -kb xd004/v1 -km narze`
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by narze** [comment]
@fauxpark Thanks for the instant review! I've verified that it works.

**Review by fauxpark — APPROVED** [review]

File: keyboards/xd004/keymaps/narze/keymap.c [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/xd004/keymaps/narze/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xd004/keymaps/narze/keymap.c (hunk 3 above)
Comment (narze) [reply keyboards/xd004/keymaps/narze/keymap.c:17]: Ok I've added the license

**Review by drashna — APPROVED** [review]
Thanks!

**Pull request merged**